draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1711,6 +1711,78 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
 </section>
 </section>
 
+<section title="Trailer Fields" anchor="trailer.fields">
+   <iref item="trailer fields"/>
+   <iref item="trailers"/>
+
+<section title="Purpose" anchor="trailers.purpose">
+<t>
+   In some cases specific to the transfer coding or HTTP version, additional
+   metadata can be sent after the initial header section has been completed
+   (during or after transmission of the payload body) that might be
+   dynamically generated while the body is being sent, such as a message
+   integrity check, digital signature, or post-processing status.
+   For example, the chunked coding in HTTP/1.1 allows a trailer section after
```

Comment:
New para?

Reply:
it's just 2 sentences and looks better after the above deleted.